doc/xml/documentation.xml

```diff
@@ -72,16 +73,16 @@ sudo usermod -aG docker testdoc
 ./doc.pl --out=html --include=user-guide --no-cache --var=os-type=debian --var=os-image=ubuntu:16.04 --var=package=test/package/pgbackrest_2.08-0_amd64.deb
         </code-block>
 
-        <p>CentOS/RHEL 7:</p>
+        <p>RHEL/CentOS 7:</p>
 
         <code-block type="bash">
-./doc.pl --out=html --include=user-guide --no-cache --var=os-type=centos7 --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
+./doc.pl --out=html --include=user-guide --no-cache --var=os-type=rhel --var=os-image=centos:7  --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
         </code-block>
 
         <p>CentOS/RHEL 8:</p>
```

Comment:
Ah, this is why the doc/README.md is still showing the CentOS/RHEL vs RHEL/CentOS....

Reply:
Also fixed in 365a558c089.

Reply:
And again in 1eb931b8ec99.